docker-compose.yaml

```diff
@@ -82,6 +82,19 @@ services:
       nofile:
         soft: 65536
         hard: 65536
+  elasticsearch8:
+    image: elasticsearch:8.6.2
+    environment:
+      http.port: 9204-9300
+      discovery.type: single-node
+      xpack.security.enabled: "false"
+      ES_JAVA_OPTS: "-Xms750m -Xmx750m" # https://github.com/10up/wp-local-docker/issues/6
+    ports:
+      - "9204:9204"
+    ulimits:
+      nofile:
+        soft: 65536
+        hard: 65536
```

Comment:
Did some tests failed because this was missing?

Reply:
Yes, the elasticsearch 8 tests were failing with `test.sh` without this.